admin-manual.md

```diff
@@ -0,0 +1,174 @@
+      <Resource name="jdbc/MISODB" type="javax.sql.DataSource"
+      driverClassName="com.mysql.jdbc.Driver"
+      initialSize="32"
+      maxIdle="10"
+      maxActive="100"
+      maxWait="1000"
+      removeAbandoned="true"
+      removeAbandonedTimeout="120"
+      logAbandoned="true"
+      testWhileIdle="true"
+      testOnBorrow="true"
+      testOnReturn="true"
+      validationQuery="select 1"
+      url="jdbc:mysql://localhost:3306/lims?autoReconnect=true&amp;zeroDateTimeBehavior=convertToNull&amp;useUnicode=true&amp;characterEncoding=UTF-8"
+      username="tgaclims"
+      password="tgaclims"/>
+    </Context>
+
+If your Tomcat install has the `autoDeploy="true"` flag set in `server.xml`, if
+you delete the `webapps/ROOT` directory and the `ROOT.war` file, Tomcat will
+delete the context `ROOT.xml` file. Either set autoDeploy to false, and
+manually deploy your webapp, or make the `ROOT.xml` file undeletable by using
+`chattr +i` (`chattr -i` will undo this operation). [Upstream
+bug](https://issues.apache.org/bugzilla/show_bug.cgi?id=40050)
+
+Set the database path:
+
+    url="jdbc:mysql://your.database.server:3306/lims"
+
+Download some supporting JARs:
+
+    cd $TOMCAT_ROOT/lib
+    curl -O https://repos.tgac.ac.uk/miso/common/mysql-connector-java-5.1.10.jar
+    curl -O https://repos.tgac.ac.uk/miso/common/jndi-file-factory-1.0.jar
+
+Append the following line to `$TOMCAT_ROOT/bin/setenv.sh` or, if using Tomcat from Debian or Ubuntu, `/etc/default/tomcat8`:
+
+    JAVA_OPTS="$JAVA_OPTS -Dsecurity.method=jdbc -Xmx768M"
+
+Create the directory `/storage/miso`.
+
+	cd /storage/miso
+    curl https://repos.tgac.ac.uk/miso/latest/miso_userspace_properties.tar.gz | tar xvfz -
+
+The configuration files are:
+
+| File                      | Purpose                                                    |
+|---------------------------|------------------------------------------------------------|
+| `issuetracker.properties` | settings for an issue tracking system, such as JIRA or RT. |
+| `mail.properties`         | email settings so that MISO can send emails to users.      |
+| `security.properties`     | properties to set the security environment (see below).    |
+| `submission.properties`   | properties to set the submission environment (see below).  |
+
+## Security Environment
+MISO can use either LDAP or SQL as an authentication mechanism. The mechanism
+is set in both `security.properties` and the `setenv.sh` file and both must be
+the same.
+
+If you are using the database (aka JDBC), set the security method to `jdbc`. The default configuration should work properly.
+
+For using LDAP, set the security method to `ldap`. Additional settings are
+needed for LDAP in the `security.properties`. Talk to your LDAP administrator.
+
+# Setting Up the Notification Server
+The notification server is a Java daemon that scans the paths containing
+sequencer output. Configuration for `systemd`-based Linux systems is provided.
```

Comment:
sequencer output. It is not required for a functioning MISO install, but is very helpful for adding sequencing Runs into MISO automatically.